components/runtime/squeak4/Makefile

```diff
@@ -0,0 +1,83 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"). You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Package Makefile for the "Classic VM" Squeak Smalltalk V4 system
+# See http://squeak.org and http://squeakvm.org
+#
+# The squeak4 vm can only run Squeak4 images; it will not work with 
+# Squeak5 images; squeak5 uses  different technology called "Cog Spur"
+#
+# squeak4 uses cmake and a custom configure script (not GNU autoconf)
+# in order to build this package, "cmake" must be installed
+#
+
+# squeak4 can be built both in 32-bit and 64-bit 
+# but generally SqueakV4 images are 32bit
+BUILD_BITS=32
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		squeak-4
+COMPONENT_VERSION=	4.16.7
+COMPONENT_SUMMARY=	The Squeak V4 Smalltalk Virtual Machine
+COMPONENT_PROJECT_URL=	http://www.squeak.org
+COMPONENT_ARCHIVE_URL=  http://squeakvm.org
+COMPONENT_FMRI=		runtime/squeak-4
+COMPONENT_CLASSIFICATION=	Development/Other Languages
+SVN_REPO=		http://squeakvm.org/svn/squeak/trunk/
+SVN_REV=		3775
+SVN_HASH=  sha256:17f565a44201dd41aa7ae65861577c76c72097bbdae8d5a2f96c5aee1e321b63
+
+# See http://wiki.squeak.org/squeak/933
+# See http://wiki.squeak.org/squeak/159
+COMPONENT_LICENSE=      MIT
+COMPONENT_LICENSE_FILE= squeak4.license
+
+TEST_TARGET= $(NO_TESTS)
+
+include $(WS_MAKE_RULES)/common.mk
+
+PATH=$(PATH.gnu)
+
+# squeak4 configure looks for .svn or svnversion file, but
+# prep-svn.mk did a svn export and created an unversioned copy of the sources
+COMPONENT_PRE_CONFIGURE_ACTION = ( echo $(SVN_REV) > $(SOURCE_DIR)/platforms/unix/svnversion )
+
+CONFIGURE_SCRIPT= $(SOURCE_DIR)/platforms/unix/cmake/configure
+CONFIGURE_OPTIONS +=	--src=$(SOURCE_DIR)/src
+
+# squeak4 installs squeak.sh and wrapper script squeak in /usr/bin
+# the squeak.sh script uses "kdialog" and we don't include it
+# rename the scripts and the manpage squeak4.sh and squeak4
+# to avoid a conflict with the squeak-5 package (mediator "squeak')
+COMPONENT_POST_INSTALL_ACTION = ( \
+  $(CP) $(PROTOUSRBINDIR)/squeak $(PROTOUSRBINDIR)/squeak4; \
```

Comment:
Likely we want $(MV) here to avoid confusing next person, looking on sample-manifest